runtime/autoload/netrw.vim

```diff
@@ -11454,24 +11454,29 @@ fun! s:NetrwLocalRmFile(path,fname,all)
 
   else
    " attempt to remove directory
+   let rmfile= substitute(rmfile,'[\/]$','','e')
    if !all
     echohl Statement
     call inputsave()
-    let ok= input("Confirm *recursive* deletion of directory<".rmfile."> ","[{y(es)},n(o),a(ll),q(uit)] ")
+    " make it be a bit harder to make mistake by inputting upper case
+    let ok= input("Confirm *recursive* deletion of directory<".rmfile."> ","[{Y(ES)},n(o),A(LL),q(uit)] ")
     call inputrestore()
-    let ok= substitute(ok,'\[{y(es)},n(o),a(ll),q(uit)]\s*','','e')
+    let ok= substitute(ok,'\[{Y(ES)},n(o),A(LL),q(uit)]\s*','','e')
     if ok == ""
      let ok="no"
     endif
-    if ok =~# 'a\%[ll]'
+    if ok =~# 'a\%[ll]'->toupper()
      let all= 1
     endif
-   endif
-   let rmfile= substitute(rmfile,'[\/]$','','e')
-
-   if all || ok =~# 'y\%[es]' || ok == ""
```

Comment:
also please note: i removed this `ok == ""` though not sure when it would happen (or mostly it would not happen), but it scared me if i just pressed 'enter' somehow.